zproject/prod_settings_template.py

```diff
@@ -38,6 +38,12 @@
 # EXTERNAL_HOST to e.g. zulip.example.com:1234 here.
 EXTERNAL_HOST = 'zulip.example.com'
 
+# If EXTERNAL_HOST is not a valid domain (e.g. if it's just an IP address),
+# you should set FAKE_EMAIL_DOMAIN below to something that is
+# a (potentially fake) domain. It will be used for email addresses
+# of bots and, if email visibility is disabled, for dummy emails of users.
+# FAKE_EMAIL_DOMAIN = 'fake-domain.example.com'
```

Comment:
Just to be clear, the reading was meant to be ""Fake email" domain" -- i.e. the domain Zulip uses to generate fake email addresses reserved for its own use.

Reply:
Yeah, I edited out that ``(potentially fake)`` parenthesis, it seems unnecessary and perhaps confusing.